src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarnings/InputSuppressWarningsHolder.java

```diff
@@ -7,13 +7,13 @@
     int a;
     @SuppressWarnings(InputSuppressWarningsHolder.UNUSED)
     int b;
-    @SuppressWarnings(com.puppycrawl.tools.checkstyle.checks.annotation.InputSuppressWarningsHolder.UNUSED)
+    @SuppressWarnings(InputSuppressWarningsHolder.UNUSED)
```

Comment:
This type of change must be reverted. We already have no package usage above. This is here for a specific test.